app/config/remote_vetting.yml

```diff
@@ -10,8 +11,21 @@ parameters:
     assertionConsumerUrl: https://selfservice.stepup.example.com/second-factor/acs
     privateKey: "%saml_sp_privatekey%"
 
+  identity_encryption_configuration:
+    encryption_public_key: |-
```

Comment:
Could we get the key through a file. I would love to use the key myself for testing purposes.